docs/team/wpinrui.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Ivan's Project Portfolio Page
+---
+
+### Project: TutorAid
+
+TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of each students' details. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
+
+Given below are my contributions to the project.
+
+* **Team Lead**
+  * As the team lead, I am in charge of ensuring that deadlines are met and project objectives are fulfilled.
+* **Documentation**
+  * I have contributed to the documentation in the following areas:
+    * User Guide: Unset payment made
+    * User Guide: Set payment made
+    * Developer Guide: Target user profile, value proposition, user stories, use cases, non-functional requirements, glossary
```

Comment:
LGTM

Reply:
Agree with this, but maybe we can also standardise the way we are recording our contributions in this page across the team before merging?

Reply:
Yes agree, we can consider standardizing the format for recording of contributions.

Reply:
LGTM, I assume we will be following this format since I saw Jun Wei and Preshita adopted this also.